examples/segmentation/image_segmentation_data_preparation.ipynb

```diff
@@ -349,7 +349,7 @@
    "source": [
     "Each input image in `raw` uses a 256 x 256 matrix of pixel _BGR_ values to define a visual picture. Each corresponding ground truth image in `mask` uses a 256 x 256 matrix of pixel _object class_ values, resulting in a pixel-by-pixel object class map. The number of classes required for an image segmentation ground truth map is equal to the number of object classes plus 1. \n",
     "\n",
-    "The image segmentation model in this example detects two object classes in input images: _player_ and _ball_. Every input image pixel BGR array corresponds to a matching ground truth pixel-wise object class array. Each ground truth image pixel contains a class value: it must be 0 (background), or 1 (player), or 2 (ball). \n",
+    "The image segmentation model in this example detects two object classes in input images: _player_ and _ball_. Every input image pixel BGR array corresponds to a matching ground truth pixel-wise object class array. Each ground truth image pixel contains a class value: it must be 0 (background), or 1 (player), or 2 (background). \n",
```

Comment:
Class value 2 should be ball instead of background.

Reply:
Class value 2 typo in image segmentation data prep example updated to 'ball'.